vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/action/TouchActions.java

```diff
@@ -130,9 +130,10 @@ public void swipeUntil(SwipeDirection direction, Duration swipeDuration, Boolean
         Duration stabilizationDuration = mobileApplicationConfiguration.getSwipeStabilizationDuration();
         int swipeLimit = mobileApplicationConfiguration.getSwipeLimit();
         BufferedImage previousFrame = null;
+        SwipeCoordinates swipeCoordinates = direction.calculateCoordinates(genericWebDriverManager.getSize());
```

Comment:
Just in case, @uarlouski  you can cover these changes with a unit test. So we don't end up with abusing calls to the driver again :)

Reply:
<img width="564" alt="Screenshot 2021-07-02 at 15 07 39" src="https://user-images.githubusercontent.com/30923336/124869679-3aee0f80-dfca-11eb-80ae-eba903a4a28b.png">

Reply:
I do not see any prerequisites to end up abusing driver calls so don't want to clutter unit tests code

Reply:
+ 1 line
verify(genericWebDriverManager).getSize() ?

Reply:
it's the first step to the cluttering of unit tests with not crucial checks, 1 line, then one more and so on